djtriggers/logic.py

```diff
@@ -147,3 +103,54 @@ def clean_triggers(expiration_dt=None, type_to_table=None):
         nr_deleted += 1
 
     return nr_deleted
+
+
+def _process_async(triggers, use_statsd):
+    """
+    Process given triggers asynchronously
+    :param django.db.models.query.QuerySet triggers: triggers to be processed
+    :param bool use_statsd: whether to use statsd
+    :return: None
+    """
+    logger.info('Start processing %d triggers of type %s asynchronously', triggers.count(), triggers[0].trigger_type)
+
+    group([trigger.process.subtask({'use_statsd': use_statsd},
+                                   max_retries=getattr(settings, 'DJTRIGGERS_CELERY_TASK_MAX_RETRIES', 0))
+           for trigger in triggers]).apply_async()
```

Comment:
I'm not sure how this works, is it possible to write this logic to be more readable so it is more clear what it does?